backend/app/api/opentripmap.py

```diff
@@ -0,0 +1,184 @@
+"""
+This script contains the logic for interacting with the OpenTripMap API using FastAPI.
+(https://rapidapi.com/opentripmap/api/places1/playground/apiendpoint_c92ce138-db30-4713-9058-752835430bc7)
+It can be seen as an extension of the events.py file.
+
+Author: Ruben Vandamme
+Date: 04 November 2024
+Usage:
+    - Call the query_opentripmap function with a name of a city to get a list of relevant Events.
+"""
+
+import requests
+from typing import List
+from app.models.event_model import Event
+from pathlib import Path
+from app.config import OPENTRIPMAP_API_KEY
+
+HOST = "opentripmap-places-v1.p.rapidapi.com"
+
+def make_request(endpoint: str, query_params: dict) -> dict:
```

Comment:
I'd use pydantic / typedDict with actual fields, not generic dicts

Reply:
the keys are not fixed as it is a generic function